test_pymole.py

```diff
@@ -32,7 +32,9 @@ def get_json(mole, url):
 
 def get_json_stream(mole, url):
     for update in mole.get(url).iter_encoded():
-        yield json.loads(update.decode('utf-8'))
+        update = update.decode('utf-8')
+        if update != '\n':
+            yield json.loads(update[5:])  # Ignore "data: " prefixed  with the json content
 
```

Comment:
maybe assert with ``update.startswith('data:')`` that you're cutting the right thing